client/src/components/KanbanItem/index.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
+import Styled from '@/components/KanbanItem/style';
+import { StoryType } from '@/types/story';
+import { useStoryDispatch } from '@/lib/hooks/useContextHooks';
+import useInput from '@/lib/hooks/useInput';
+import { updateStoryWithName } from '@/lib/api/story';
 
-const KanBanItem = () => {
-  return <div></div>;
+type KanbanItem = {
+  story: StoryType;
+  storyObj: StoryType;
+  setDeleteKey(arg: number): void;
+  setShowModal(arg: boolean): void;
 };
 
-export default KanBanItem;
+const KanbanItem = ({ story, storyObj, setDeleteKey, setShowModal }: KanbanItem) => {
+  const { key, value, onChange } = useInput('');
+  const useDispatch = useStoryDispatch();
```

Comment:
```suggestion
  const storyDispatch = useStoryDispatch();
```
로 변경하는 건 어떨까요?? 반환 받은 dispatch가 훅이 아닌 단순 dispatch라서요 ㅎㅎ